### ISSUE TITLE
[Help wanted] We need donation to make cdnjs better.

### ISSUE THREAD

**Issue by PeterDaveHello** [cdnjs/cdnjs#4605]
Dear all,

As you know we are opensource project and provide service for free, only depends on our sponsors like Cloudflare and Algolia, and now we think we'll need some donation so that we can make the service even better, here are some important reason that why we need donation:
1. There are many issues need to be fixed but we don't have enough human resource to make it done, it'll be good if we can pay or donate the contributors or even pay for the issues, maybe some of you know that we paid bitcoin for make libs to use auto-update before, just like that.
2. We rent our build server on DigitalOcean before(a 4 cores and 8GB ram server, cost 80$ per month), it actually made us some cost, now I'm still a student study at school, and for the opensource project, I'll be happy and also I can use the server at my school to help us build/develop this project, but I won't be student or can use the server at school forever, the better solution is still to rent a server at DigitalOcean.

In summary, there are some good services we may need, so that we can make this project better, and we'll need help!

Currently, we have two donation channels:
[tip4commit](https://tip4commit.com/github/cdnjs/cdnjs)
[bountysource](https://www.bountysource.com/teams/cdnjs/bounties?utm_source=cdnjs&utm_medium=shield&utm_campaign=bounties_posted)

Any donations or welcome, as the largest opensource and public web lib cdn, we have only few people on this project andreally need your help, join us to make our service better!

Best,
Peter

**Comment by PeterDaveHello** [comment]
cc @thomasdavis

**Comment by tomkwok** [comment]
> There is a awesome, valuable service tinypng which can make the png images smaller but with almost the origin quality(believe me, I can not recognize which one is minified), and we want to use this service to make the png images on cdnjs smaller, so the cdn service can be even better!

@PeterDaveHello 

Tinypng is not awesome, nor valuable. It's basically an online PNG quantizer with an API. pngquant, an open-source command line tool, does exactly the job. https://pngquant.org/

I think a CDN service shouldn't do lossy compression on image resources in the first place, but rather, lossless compression of PNG files using optipng or other tools. Refer to http://css-ig.net/png-tools-overview.html.

**Comment by PeterDaveHello** [comment]
@tomkwok Yeah you are right, I forgot to update this issue, we only use optipng to do lossless compress :)

**Comment by tomkwok** [comment]
@PeterDaveHello And I have one question to ask. https://github.com/cdnjs/cdnjs/issues/3638 is totally confusing. How can I request a new library to be added to the cdnjs repo now? From the commit logs, it seems like you are manually adding new libraries or doing version updates of existing libraries.

**Comment by PeterDaveHello** [comment]
https://github.com/cdnjs/cdnjs#enabling-gitrecommended-or-npm-auto-update
> cdnjs automatically updates libraries that are known to be hosted on npm or git repo, e.g., Lodash. npm auto-update relys on each release and git auto-update relys on the tags in git repo. This auto-update script runs every hour, but the update result won't be committed until one of our maintainers audit the updates, because many libs will change the naming or directory structure during different versions, and we may need to minify the lib without pre-minified dist files, so it'll be reasonable to delay for at most 30 hours, if you think there is a lib didn't been updated, please wait at least 30 hours for the process to audit it.
I think you should better open a new issue to ask/discuss an un-related question.
